### PR TITLE
treewide: bump copyright year to 2021 in generated files

### PR DESCRIPTION
**Pull request by tklauser**

### CODE REVIEW

**Comment by tklauser** [comment]
test-me-please

**Review by aanm — APPROVED** [review]

**Review by qmonnet — APPROVED** [review]
Thanks!

**Review by pchaigno — APPROVED** [review]
I intend to merge this once CI is green, without waiting for all reviews.

**Comment by tklauser** [comment]
retest-net-next

**Pull request merged**
